.gitignore

```diff
@@ -5,7 +5,7 @@
 *.pyd
 *.cpp
 *.so
-build
+build*
```

Comment:
This change is unrelated to this PR.

Reply:
ah... I used different build folders for running on different cuda versions, that's the reason. Will revert it back

Reply:
Also this line? https://github.com/cupy/cupy/pull/4281/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R25

Reply:
😅 no idea how that came up! will fix right away!